homeworks/andrii.morozov_AndriiMorozov88/GamesProject/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>About Me</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <main class="main">
+    <h1>About Me</h1>
+    <p class="main__paragraf">Hi! My name is Andrii Morozov and I'm a Junior Frontend Developer. I am already familiar with main Web Technologies like HTML, CSS, JavaScript and Git version control system.</p>
+    <p class="main__paragraf">This page was developed during the course <a href="https://www.mastersacademy.education/frontend-for-beginners-it" class="link link--red" target="_blank">'Frontend for beginners' from Masters Academy in 2023.</a></p>
+    <p class="main__paragraf">This is a social project from MOCG company where I got an opportunity to work with Frontend mentors and to create my own small project for the portfolio. </p>
+    <p class="main__paragraf">You can contact me via <a href="https://www.facebook.com/profile.php?id=100024573128932" class="link link--blue" target="_blank">facebook</a> and/or check out my GitHub.</p>
```

Comment:
Please use similar style for all links on the page (`.link--red`)
Also, the link on the design do have an underline